content/src/repository/extensions/DeploymentsRepository.ts

```diff
@@ -176,7 +185,24 @@ export class DeploymentsRepository {
     return `(${equalWithEntityIdComparison} OR ${timestampComparison})`
   }
 
-  getSnapshot(
+  getFullSnapshot(): Promise<FullSnapshot[]> {
+    return this.db.map(
+      `SELECT entity_id, entity_type, entity_pointers, auth_chain, date_part('epoch', local_timestamp) * 1000 AS local_timestamp ` +
+        `FROM deployments ` +
+        `WHERE deleter_deployment IS NULL ` +
+        `ORDER BY local_timestamp DESC, LOWER(entity_id) DESC`,
```

Comment:
```suggestion
        `ORDER BY local_timestamp ASC, LOWER(entity_id) DESC`,
```

Reply:
Okay, but shouldn't we start with latest deployments so we can ignore the old ones assuming that we already processed them?